packages/@aws-cdk/aws-route53-patterns/lib/website-redirect.ts

```diff
@@ -107,4 +102,58 @@ export class HttpsRedirect extends Construct {
       new AaaaRecord(this, `RedirectAliasRecordSix${hash}`, aliasProps);
     });
   }
+
+  /**
+   * Gets the stack to use for creating the Certificate
+   * If the current stack is not in `us-east-1` then this
+   * will create a new `us-east-1` stack.
+   */
+  private get certificateScope(): Construct {
+    const stack = Stack.of(this);
+    const parent = stack.node.scope;
+    if (!parent) {
+      throw new Error(`Stack ${stack.stackId} must be created in the scope of an App or Stage`);
+    }
+    if (Token.isUnresolved(stack.region)) {
+      throw new Error(`When ${ROUTE53_PATTERNS_USE_CERTIFICATE} is enabled, a region must be defined on the Stack`);
+    }
```

Comment:
whoah, what is this? In the PR description, you said that there's no breaking change, but what about scenarios where you're using `DnsValidatedCertificate` without a region? Wouldn't we throw an error when we move it to `Certificate`?

Reply:
Why is that a breaking change? We are not breaking any existing user since we'll only move to `Certificate` when you enable the feature flag.

Reply:
Yeah those users would have to provide a region, which I don't think is too much to ask when you are going cross region stuff. The alternative is that we make an assumption

1. Assume that they are in `us-east-1` and if they are not the error is thrown by CloudFormation
2. Assume that they are not in `us-east-1` and always create a support stack. We can always switch this later if this is the direction users want us to go.